docs/docs/index.md

```diff
@@ -141,14 +141,14 @@ to restart the terminal session to take effect.
 
 #### For Mac and Linux users
 
-The command to change the environment variables can be produced by `pdm --pep582 [<SHELL>]`. If `<SHELL>`
-isn't given, PDM will pick one based on some guesses.
+The command to change the environment variables can be printed by `pdm --pep582 [<SHELL>]`. If `<SHELL>`
+isn't given, PDM will pick one based on some guesses. You can run `eval "$(pdm --pep582)"` to run the command.
 
-You may want to write a line in your `.bash_profile`(or similar profiles) to make it effective when login.
+You may want to write a line in your `.bash_profile`(or similar profiles) to make it effective when logging in.
 For example, in bash you can do this:
 
 ```bash
-pdm --pep582 >> ~/.bash_profile
+eval "$(pdm --pep582)" >> ~/.bash_profile
```

Comment:
That's not correct. Users should either put `eval "$(pdm --pep582)"` or the output of `pdm --pep582` (what the original example does) into their Bash profile.

Reply:
Oops, my mistake. Yes, thanks for correcting me. I will change this part back.

Reply:
Ok, reverted! Let me know if there's anything else that needs fixing. This PR may not even be worth merging given how small it is 😅